tasks/push-rpm-data-to-pyxis/push-rpm-data-to-pyxis.yaml

```diff
@@ -29,19 +29,19 @@ spec:
       type: string
       description: The maximum number of images to be processed at once
       default: 4
+    - name: downloadedSbomPath
```

Comment:
I would recommend to use a pattern we use elsewhere which is to derive the path from some other path which is provided as a parameter - in this case `pyxisJsonPath`. And then emit that as a result, so that other tasks can use it (and it also kind of defines the dependency - that this the other task relies on this task).

So instead of having this param, you will have a result.

Reply:
The SBOM files should have its own directory, so non-SBOMs are not processed by the update script. But I changed the task to output the dir as a result.

Reply:
Yep, that's exactly what I meant, thanks.